pkg/onchain/website/uploader.go

```diff
@@ -112,6 +113,60 @@ func upload(client *node.Client, addr []byte, chunks []string, wallet *wallet.Wa
 	return &operations, nil
 }
 
+//nolint:lll
+func UploadMissedChunks(atAddress string, content string, wallet *wallet.Wallet, missedChunks string) (*[]string, error) {
+	const blockLength = 260000
```

Comment:
I think this variable should be a global constant value since it is also used [in the Upload function](https://github.com/massalabs/thyra/blob/dea904f1db69c777b292e4de908381daac468403/pkg/onchain/website/uploader.go#L57).

Reply:
ok !